about.markdown

```diff
@@ -1,18 +1,73 @@
+
+
+## How we are trying to solve this problem
+
+**We make coding social** by creating a communicative and cooperative atmosphere around it. Our workshops kick off with an installation get-together and are provided with food that attendees bring along to share.
+
+By teaching women in groups of two, we create an **environment without pressure or competition** which diminishes the fear of asking “stupid” questions.
+
+**We connect women with each other, with mentors / coaches and role models** who have shown that it doesn’t need an academic background in computer science or be good at Maths to become a successful developer.
+
+**We foster continued learning** by encouraging attendees to join project groups after the workshop. These groups are led by experienced developers and stretch over several weeks to months. In addition to that, learners of all levels come together regularly on our “Code & Cake”.
+
+
+## What do we mean by „women“?
+
+We welcome people who *identify as women* of *any ethnicity and background above the age of 16*. No matter if you’re still in high school, have been working for many years or are staying home to take care of relatives. Diversity matters to us, so if you are not fluent in English, don’t own a laptop, can’t find someone to care of your children during the workshop, please contact us and we will try to find a solution.
+
+## Who we are
+
+Code curious Berlin events are organized by a small group of organisers and a huge group of programmers. Everyone is volunteering for this project, because we believe that women are the future of programming. The overwhelmingly positive feedback we are receiving from our attendees and coaches is convincing us to keep this project going.
+
+**How you can help**  
+We are always looking for people to support code curious by either coaching or sponsoring our events or helping us organise them. If you want to get involved, then please drop us a line [contact@codecurious.org](mailto:contact@codecurious.org)
+
+## Team
+
+[![](../assets/images/ferdous.jpg)](https://twitter.com/ferbsx)
+
+### [Ferdous](https://twitter.com/ferbsx)
+
+Bioinformatician, Ruby Monsta, loves to travel, code and try new things.  
+
+Responsible for: Website, Event Planning, Finances & Sponsors, Attendee Coordination, Social Media
+
+[![](../assets/images/kaja.jpg)](https://twitter.com/alizenero)
+
+
+### [Kaja](https://twitter.com/alizenero)
+
+Philosopher, soccer player, Ruby coder and feminist. Loves food and her bicycle.  
+
+Responsible for: Attendee Coordination, Event Planning, Newsletter and Social Media
+
+### [Amr](https://twitter.com/amrAbdelwahab)
+### [Yonatan](https://twitter.com/shushugah)
```

Comment:
I'm fine with having individual descriptions, but I think boxing ourselves into different roles is wrong. We should be generalists, not specialists and rotate anyways.